crates/nu-protocol/src/value/primitive.rs

```diff
@@ -75,6 +78,28 @@ impl Primitive {
         }
     }
 
+    // FIXME: This is a bad name, but no other way to differentiate with our own Duration.
+    pub fn into_chrono_duration(self, span: Span) -> Result<chrono::Duration, ShellError> {
+        match self {
+            Primitive::Duration(duration) => {
+                // FIXME: Define const for the NANOS_PER_SEC?
```

Comment:
:+1: on defining some helpful constants

Reply:
Is there a specific file where I should define it?

Reply:
Putting it in the same file is fine for now.